ibis/expr/api.py

```diff
@@ -1523,6 +1523,8 @@ def convert_base(arg, from_base, to_base):
 }
 
 
+bit_xor = _agg_function('bit_xor', ops.BitXor, True)
```

Comment:
This should be potentially confusing as bit_xor can also be think of a elementwise operation between two columns. for example:

http://spark.apache.org/docs/latest/api/python/pyspark.sql.html?highlight=ntile#pyspark.sql.Column.bitwiseXOR

C++ also defines bit_xor as an operation that takes two value return compute bit_xor on them, which is similar to what bitwiseXOR in Spark.

Is there any API references that treats bit_xor as an aggregation function?

Reply:
As mentioned in https://github.com/ibis-project/ibis/issues/2641 [BigQuery](https://cloud.google.com/bigquery/docs/reference/standard-sql/aggregate_functions#bit_xor), [MySQL](https://dev.mysql.com/doc/refman/5.7/en/aggregate-functions.html#function_bit-xor), [Spanner](https://cloud.google.com/spanner/docs/aggregate_functions?hl=da#bit_xor), and [Sybase](http://infocenter.sybase.com/help/index.jsp?topic=/com.sybase.help.sqlanywhere.12.0.1/dbreference/bit-xor-function.html) all provide a `bit_xor` aggregate function.

I'm okay with picking a different name to indicate that it's an aggregate, but went with `bit_xor` as it's the same in all of those systems.

Reply:
Also called BIT_XOR in Veritca https://www.vertica.com/docs/9.2.x/HTML/Content/Authoring/SQLReferenceManual/Functions/Aggregate/BIT_XOR.htm